Gemfile

```diff
@@ -7,6 +7,7 @@ gemspec
 gem 'kaminari', github: 'jcoyne/kaminari', branch: 'sufia'
 gem 'sufia-models', path: './sufia-models'
 gem 'slop', '~> 4.2' # This just helps us generate a valid Gemfile.lock when Rails 4.2 is installed (which requires byebug which has a dependency on slop)
+gem 'active-fedora', github: 'projecthydra/active_fedora', branch: '9.7.x-stable'
```

Comment:
@awead sure!  That would be 9.7.2 since 9.7.1 was released in January. Do you have the desire?